src/tasks/daily/daily_battle_mixin.py

```
@@ -61,6 +61,48 @@ class DailyBattleMixin(MapMixin, ZipLineMixin, BattleMixin, Common):
+        # 组合模式下不会执行 BattleMixin.__init__，这里补齐战斗运行时基础属性。
+        self.last_no_number_action_time = 0
+        self.exit_check_count = 0
+        self.lv_regex = re.compile(r"(?i)lv|\d{2}")
+        self.config_type["技能释放"] = {
+            "type": "multi_selection",
+            "options": ["1", "2", "3", "4"],
+        }
+        self.config_description.update({
+            "技能释放": (
+                "勾选要自动循环释放「战技」的角色编号。\n"
+                "按编号从小到大的顺序依次释放，至少勾选一个。"
```

Ruff (0.15.13)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)